bracket.ts

```diff
@@ -1,19 +1,11 @@
 export enum Bracket {
-    UNKNOWN = -1,
-    TOTAL = 0,
-    DEAD = 1,
-    LOW = 2,
-    MEDIUM = 3,
-    HIGH = 4,
-    PRIME = 5
```

Comment:
Please revert the spacing changes and update your IDE to use the project standard of 4 spaces